src/corehost/cli/nethost/nethost.h

```diff
@@ -63,4 +63,4 @@ extern "C" NETHOST_API int NETHOST_CALLTYPE get_hostfxr_path(
     size_t * buffer_size,
     const char_t * assembly_path);
 
-#endif // __NETHOST_H__
\ No newline at end of file
+#endif // __NETHOST_H__
```

Comment:
Is this a style thing, or is the compiler actually complain about missing EOL?

Reply:
This is the `newline-eof` warning: https://github.com/llvm-mirror/clang/blob/master/test/Lexer/newline-eof.c

Reply:
ok- I'll learn to live with that :-)